app/models/search_builder.rb

```diff
@@ -93,6 +93,15 @@ def add_left_anchored_title(solr_parameters)
     solr_parameters[:q] = augmented_solr_q
   end
 
+  # remove the subject specialists facet from the config to prevent error
```

Comment:
i'd like to see some additional comments here - what error is being prevented, and under what conditions will this method transform the params?

Reply:
for the record, I'm going to merge this as-is and followup on this comment in a subsequent commit.